src/app/pages/confirm-test-details/components/confirm-submit-modal/__mocks__/confirm-submit-modal.mock.ts

```diff
@@ -0,0 +1,12 @@
+import { DateTime } from '@shared/helpers/date-time';
+import { SlotDetail } from '@dvsa/mes-journal-schema';
+
+export class JournalEarlyStartModalMock {
```

Comment:
A very similar mock of this already exists here ```src/app/pages/journal/components/journal-early-start-modal/__mocks__/journal-early-start-modal.mock.ts```

Can we just re-use this mock?